sql/core/src/main/scala/org/apache/spark/sql/SparkSession.scala

```diff
@@ -848,6 +851,17 @@ object SparkSession {
     }
 
     /**
+     * Inject extensions into the [[SparkSession]]. This allows a user to add Analyzer rules,
+     * Optimizer rules, Planning Strategies or a customized parser.
+     *
+     * @since 2.3.0
```

Comment:
In the JIRA, the target version is 2.2. Do we still plan to backport it to 2.2.0?